resources/schema.json

```diff
@@ -44,11 +44,15 @@
         },
         "cran": {
           "type": "string",
-          "description": "[FUTURE RELEASE], Packages to install from CRAN (RStudio Workbench only)"
+          "description": "List of packages to install from CRAN, each separated by a space(RStudio Server only)"
```

Comment:
Should we really be removing the `[FUTURE RELEASE]` here? No idea when we'll have support for any of these extra package types.